### PR TITLE
Flux get started intro and data model

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/get-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+{{< /code-tabs-wrapper >}}
+
+Each example includes the following:
+
+1. A `from()` function to retrieve data from the data source.
+2. `range()`, `filter()`, or both to filter data based on column values.
+3. A `mean()` function to calculate the average of values returned from the data source.
+4. A `yield()` function to yield results to the user.
+
+Flux is designed to retrieve data from data source and pipe it through a
```
Comment (pierwill) [inline content/flux/v0.x/get-started/_index.md:86]:
```suggestion
Flux is designed to retrieve data from data source and send it through a
```

File: content/flux/v0.x/get-started/data-model.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+description: >
+  ...
+menu:
+  flux_0_x:
+    name: Data model
+    parent: Get started
+weight: 101
+---
+
+To fully understand how to use Flux to process your data, you must understand
```
Comment (pierwill) [inline content/flux/v0.x/get-started/data-model.md:12]: Another option:

```suggestion
To get the most out of using Flux to process your data, you must understand
```

File: content/flux/v0.x/get-started/data-model.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+- [Group key](#group-key) 
+
+{{< flux/data-model >}}
+
+#### Stream of tables
+A **stream of tables** is a collection of zero or more [tables](#table).
+Queried data sources return results as a stream of tables.
+
+#### Table
+A **table** is a collection of [columns](#column) partitioned by the [group key](#group-key).
```
Comment (pierwill) [inline content/flux/v0.x/get-started/data-model.md:29]:
```suggestion
A **table** is a collection of [columns](#column) partitioned by [group key](#group-key).
```

File: content/flux/v0.x/get-started/data-model.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+#### Row
+A **row** is a collection of associated [column](#column) values.
+
+#### Group key
+A **group key** defines which column values to include in a table.
+All rows in a table contain the same values in group key columns.
+All tables in a stream of tables have a unique group key, but group key
+modifications are applied to a stream of tables.
+
+##### Group key examples
```
Comment (pierwill) [inline content/flux/v0.x/get-started/data-model.md:44]: Maybe:

```suggestion
##### Example group keys
```

File: content/flux/v0.x/get-started/data-model.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+Group keys contain key value pairs where each key represents a column name and
```
Comment (pierwill) [inline content/flux/v0.x/get-started/data-model.md:45]:
```suggestion
Group keys contain key-value pairs, where each key represents a column name and
```

File: content/flux/v0.x/get-started/data-model.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+  |> group(columns: ["foo", "bar"], mode: "by")
+```
+
+### Table grouping example
+The tables below represent data returned from InfluxDB with the following schema:
+
+- `example` measurement
+- `loc` tag with two values
+- `sensorID` tag with two values
+- `temp` and `hum` fields
```
Comment (pierwill) [inline content/flux/v0.x/get-started/data-model.md:100]: These lines seem a little sparse...

**Review by pierwill — APPROVED** [review]
Looks good! Have made a few suggestions.

File: assets/styles/layouts/article/_flux.scss [hunk 7]
```diff
@@ -103,4 +103,197 @@ h2,h3,h4 {
     }
   }
   
+  pre#group-by-example {
```
[the remark this answers is not in the document]
Comment (sanderson) [reply assets/styles/layouts/article/_flux.scss:106]: Yes, it is.

File: content/flux/v0.x/get-started/data-model.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+#### Column
+A **column** is a collection of values of the same [basic type](#)
+that contains one value for each [row](#row).
+
+#### Row
+A **row** is a collection of associated [column](#column) values.
+
+#### Group key
+A **group key** defines which column values to include in a table.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/get-started/data-model.md:39]: If the column is in the group key, all the column values in the table will be the same. The group key is the driving force for how tables are structured; not the other way around.

File: content/flux/v0.x/get-started/data-model.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+- [Table](#table)
+- [Column](#column)
+- [Row](#row)
+- [Group key](#group-key) 
+
+{{< flux/data-model >}}
+
+#### Stream of tables
+A **stream of tables** is a collection of zero or more [tables](#table).
+Queried data sources return results as a stream of tables.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/get-started/data-model.md:26]: "Flux queries" don't always return a stream of tables; "Data sources" (InfluxDB, CSV, MySQL, etc.) return a stream of tables.

**Pull request merged**